AppleMusic-SwiftUI/Radio/RadioStations/RadioStationsModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RadioStationsModel.swift
+//  AppleMusic-SwiftUI
+//
+//  Created by Алексей Кудинов on 12.09.2022.
+//
+
+import SwiftUI
+
+struct RadioStationsModel: Hashable {
+    let image: String
+    let title: String
+    let subtitle: String
+}
+
+struct radioStationCatalogue {
```

Comment:
Название объектов с большой буквы
```suggestion
struct RadioStationCatalogue {
```